pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -633,37 +635,67 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 		switch t := v.(type) {
 		case *extensions.Deployment:
 			//delete deployment
-			rpDeployment, err := kubectl.ReaperFor(extensions.Kind("Deployment"), client)
+			label := labels.SelectorFromSet(labels.Set(map[string]string{"io.kompose.service": t.Name}))
```

Comment:
`io.kompose.service` is used in a lot of places. maybe we could use `transformer.ConfigLabels` to handle all the labels.
Or other option would be to create new function or constant that will hold that string